content/build/hydrobuild.md

```diff
@@ -54,58 +54,103 @@ Docker Desktop 4.23.0 and later versions ship with a Hydrobuild-compatible
+Enable the [Builds view](../desktop/use-desktop/builds.md) in Docker Desktop
+and complete the following steps:
 
-   This creates a builder named `cloud-<org>-default`.
+1. Sign in to your Docker account using the **Sign in** button in Docker Desktop.
+
+2. Open the Docker Desktop settings and navigate to the **Builders** tab.
+
+3. Under **Available builders**, select **Create builder**.
+
+   ![Create builder GUI screenshot](./images/create-builder-gui.webp)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+The builder has native support for the `linux/amd64` and `linux/arm64`
+architectures. This gives you a high-performance build cluster for building
+multi-platform images natively.
 
 ## Use Hydrobuild from the CLI
 
-To run a build using Hydrobuild, invoke a build command and specify the
-name of the builder using the `--builder` flag.
+To run a one-time build using Hydrobuild, invoke a build command and specify
+the name of the builder using the `--builder` flag.
 
 ```console
 $ docker buildx build --builder cloud-<org>-default --tag <org>/<image> .
 ```
 
-> **Note**
->
-> Building with `--tag` loads the build result to the local image store
-> automatically when the build finishes. To build without a tag and load the
-> result, you must pass the `--load` flag.
->
-> If you use the containerd image store, you must always pass `--load` to
-> download the results, even if you build with a tag.
+If you want to use Hydrobuild without having to specify the `--builder` flag
+each time, you can set it as the default builder.
+
+{{< tabs group="ui" >}}
+{{< tab name="CLI" >}}
+
+Run the following command:
+
+```console
+$ docker buildx use cloud-<org>-default --global
+```
+
+{{< /tab >}}
+{{< tab name="GUI" >}}
+
+1. Open the Docker Desktop settings and navigate to the **Builders** tab.
+2. Find the Hydrobuild builder under **Available builders**.
+3. Open the drop-down menu and select **Use**.
 
-To build a multi-platform image and push it to a registry:
+   ![Selecting Hydrobuild as default using the GUI](./images/set-default-builder-gui.webp)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+Changing your default builder with `docker buildx use` only changes the default
+builder for the `docker buildx build` command. The `docker build` command still
+uses the `default` builder, unless you specify the `--builder` flag explicitly.
+
+If you use build scripts, such as `make`, we recommend that you update your
+build commands from `docker build` to `docker buildx build`, to avoid any
+confusion with regards to builder selection. Alternatively, you can run `docker
+buildx install` to make the default `docker build` command behave like `docker
+buildx build`, without discrepancies.
+
+## Loading build results
+
+Building with `--tag` loads the build result to the local image store
+automatically when the build finishes. To build without a tag and load the
+result, you must pass the `--load` flag.
+
+Loading the build result for multi-platform images is not supported. Use the
```

Comment:
I guess we don't want to mention that it is supported with containerd enabled here?

Reply:
Unfortunately it's not supported either :/

```
ERROR: oci for multi-node builds currently not supported
```